offer-manager-domain/src/main/groovy/life/qbic/business/offers/Currency.groovy

```diff
@@ -11,15 +11,22 @@ import java.text.NumberFormat
  */
 class Currency {
 
-    private final static NumberFormat numberFormat = NumberFormat.getInstance(Locale.US)
+    final static String SYMBOL = "€"
+    private final static NumberFormat NUMBER_FORMAT = NumberFormat.getInstance(Locale.US)
+
+    static String format(double value) {
+        DecimalFormat decimalFormat = getFormatterWithoutSymbol()
+        return decimalFormat.format(value)
+    }
```

Comment:
Why did you need to change that here? It worked before on my machine. Was it an issue at your side or is it just a "cosmetic" change (if so why)?

Reply:
By delegating the decision on the symbol away from here we can change the currency more flexible. The main point was however that this forces all currencies to be formatted equally. Previously, a change in the formatterWithoutCurrency was not reflected in the formatterWithCurrency.